library/VTKExtensions/Rendering/vtkF3DCachedLUTTexture.cxx

```diff
@@ -51,13 +56,18 @@ void vtkF3DCachedLUTTexture::Load(vtkRenderer* ren)
     vtkImageData* img = reader->GetOutput();
     int dims[3];
     img->GetDimensions(dims);
+    if (dims[0] != dims[1])
+    {
+      vtkWarningMacro("LUT cache has unexpected dimensions");
```

Comment:
should we have an assert here instead?

Reply:
nope, this case can happen if a user starts manually puting cache files in the cache path